tests/providers/google/cloud/operators/test_local_to_gcs.py

```diff
@@ -42,15 +44,24 @@ def setUp(self):
             'start_date': datetime.datetime(2017, 1, 1)
         }
         self.dag = DAG('test_dag_id', default_args=args)
+        self.testfile1 = tempfile.NamedTemporaryFile(delete=False)
+        self.testfile1.write(b"x" * 393216)
+        self.testfile1.flush()
+        self.testfile2 = tempfile.NamedTemporaryFile(delete=False)
+        self.testfile2.write(b"x" * 393216)
+        self.testfile2.flush()
+        self.testfiles = [self.testfile1.name, self.testfile2.name]
```

Comment:
Are those files deleted after test?

Reply:
These are created in `/tmp` directory so I I figure they would be ok per earlier feedback. Looks like the tests are passing now.

However, I realized that physical files aren't even needed in the tests so I'm going to try with dummy files as strings instead.

Reply:
Actually, we can't use dummy files as strings since this now uses glob. We need to be able to test glob functionality with actual files. To accommodate, I've added a `tearDown` method to the tests to remove the files once the tests are completed.

I've also changed back to manually created named files so the wildcard searches can return more consistent results. 

Sorry the confusion, this is now passing all the checks and ready for review.

Reply:
I stand corrected, this is failing one of the mysql checks :disappointed: Will take another look tomorrow